src/implementations/environments.jl

```diff
@@ -147,6 +148,30 @@ end
 get_reward(env::RewardOverriddenEnv, args...; kwargs...) =
     foldl(|>, env.processors; init = get_reward(env.env, args...; kwargs...))
 
+
+#####
+# MaxTimeoutEnv
+#####
+
+mutable struct MaxTimeoutEnv{E<:AbstractEnv} <: AbstractEnv
+    env::E
+    max_t::Int
+    current_t::Int
+end
+
+function (env::MaxTimeoutEnv)(args...; kwargs...)
+    env.env(args...; kwargs...)
+    env.current_t = env.current_t + 1
+end
```

Comment:
Should I return `env.env` in the end? Right now `env.current_t` is returned.

Reply:
It's not required.

Reply:
Ok